matrix_client/room.py

```diff
@@ -345,3 +345,13 @@ def add_room_alias(self, room_alias):
             return True
         except MatrixRequestError:
             return False
+
+    def get_joined_members(self):
```

Comment:
Might be useless for future handling, but that's pretty basic information we need to be able to provide to the user.

> This function would be more useful to return a userid, or better yet just return a dictionary containing both.
> 
> Hmm, probably need a member class in the future to hold all this information

I agree, I'll go with the dictionary for now.